src/applications/health-care-questionnaire/questionnaire/config/submitForm.js

```diff
@@ -0,0 +1,76 @@
+// import { submitToUrl } from 'platform/forms-system/src/js/actions';
+
+const submit = (form, formConfig) => {
+  const body = formConfig.transformForSubmit(formConfig, form);
+
+  const eventData = {};
+  // console.log('submitting', { body, url: formConfig.submitUrl });
+  // just for MVP until we have an API set up
+  // eslint-disable-next-line no-console
+  console.log('calling to api', {
+    body,
+    url: formConfig.submitUrl,
+    trackingPrefix: formConfig.trackingPrefix,
+    eventData,
+  });
+  return new Promise((resolve, _reject) => {
+    resolve(body);
+  });
+  // return submitToUrl(
+  //   JSON.stringify(body),
+  //   formConfig.submitUrl,
+  //   formConfig.trackingPrefix,
+  //   eventData,
+  // );
+};
+
+const createAnAnswer = valueString => ({ valueString });
+
+const createAnswerArray = value => (value ? [createAnAnswer(value)] : []);
+
+const transformForSubmit = (_formConfig, form) => {
+  // console.log({ formConfig, form });
```

Comment:
can we clean these commented out console log up if we don't need them?

Reply:
Hey Tim! I'll look through the feedback but this is still very much a work in progress and still a draft PR. Thank you for your comments, and I'll look into your comments when I'm back in the office

Reply:
oh apologies! I didn't see the draft setting